Sources/apm-agent-ios/Agent.swift

```diff
@@ -8,9 +8,16 @@ import TrueTime
 
 public class Agent {
     
-    public static func start(with configuaration: AgentConfiguration) {
+    
+    
+    public static func start(with configuaration: AgentConfiguration, _ instrumentationConfiguration: InstrumentationConfiguration = InstrumentationConfiguration()) {
```

Comment:
Just for my understanding: does this mean that `instrumentationConfiguration` has a default value / instance and that the `start()` method can be also used without explicitly providing an `instrumentationConfiguration` object?
IMHO, that would be the desired behavior.

Reply:
correct! This change wont affect customers that used the previous start call.